subworkflows/local/functional_annotation.nf

```diff
@@ -4,24 +4,27 @@ include { DIAMOND_RHEACHEBI                               } from '../../modules/
 include { CAT_CAT as CONCATENATE_EGGNOGMAPPER_ANNOTATIONS } from '../../modules/nf-core/cat/cat/main'
 include { CAT_CAT as CONCATENATE_INTERPROSCAN_TSV         } from '../../modules/nf-core/cat/cat/main'
-include { CAT_CAT as CONCATENATE_HMMSARCH_TBLOUT          } from '../../modules/nf-core/cat/cat/main'
+include { CAT_CAT as CONCATENATE_DBCAN_OVERVIEW           } from '../../modules/nf-core/cat/cat/main'
+include { CAT_CAT as CONCATENATE_HMMSEARCH_TBLOUT         } from '../../modules/nf-core/cat/cat/main'
+// TODO: this is temporal to create a bgzip and index for the goslim summaries
+include { TABIX_BGZIP as TABIX_BGZIP_GO                   } from '../../modules/nf-core/tabix/bgzip/main'
+include { TABIX_BGZIP as TABIX_BGZIP_GOSLIM               } from '../../modules/nf-core/tabix/bgzip/main'
 
 /* EBI-METAGENOMICS */
 include { INTERPROSCAN                             } from '../../modules/ebi-metagenomics/interproscan/main'
 include { EGGNOGMAPPER as EGGNOGMAPPER_ORTHOLOGS   } from '../../modules/ebi-metagenomics/eggnogmapper/main'
 include { EGGNOGMAPPER as EGGNOGMAPPER_ANNOTATIONS } from '../../modules/ebi-metagenomics/eggnogmapper/main'
-include { GENOMEPROPERTIES                         } from '../../modules/ebi-metagenomics/genomeproperties/main'
 include { DBCAN                                    } from '../../modules/ebi-metagenomics/dbcan/dbcan/main'
-
 include { GOSLIM_SWF                               } from '../../subworkflows/ebi-metagenomics/goslim_swf/main'
 
 /* LOCAL */
-include { CONCATENATE_INTERPROSCAN_GFFS           } from '../../modules/local/concatenate_interproscan_gffs'
-include { PFAM_SUMMARY                            } from '../../modules/local/pfam_summary'
-include { INTERPRO_SUMMARY                        } from '../../modules/local/interpro_summary'
-include { HMMER_HMMSCAN as HMMSCAN_KOFAMS         } from '../../modules/local/hmmer/hmmscan/main'
-include { KEGG_ORTHOLOGS_SUMMARY                  } from '../../modules/local/kegg_orthologs_summary'
-include { KEGGPATHWAYSCOMPLETENESS                } from '../../modules/ebi-metagenomics/keggpathwayscompleteness/main'
+include { CONCATENATE_GFFS as CONCATENATE_INTERPROSCAN_GFFS } from '../../modules/local/concatenate_gffs'
+include { CONCATENATE_GFFS as CONCATENATE_DBCAN_GFFS        } from '../../modules/local/concatenate_gffs'
+include { PFAM_SUMMARY                                      } from '../../modules/local/pfam_summary'
+include { INTERPRO_SUMMARY                                  } from '../../modules/local/interpro_summary'
+include { HMMER_HMMSEARCH as HMMSEARCH_KOFAMS               } from '../../modules/nf-core/hmmer/hmmsearch/main'
+include { KEGG_ORTHOLOGS_SUMMARY                            } from '../../modules/local/kegg_orthologs_summary'
+include { KEGGPATHWAYSCOMPLETENESS                          } from '../../modules/ebi-metagenomics/keggpathwayscompleteness/main'
```

Comment:
you need to be careful with that script https://github.com/EBI-Metagenomics/assembly-analysis-pipeline/blob/5d931ea7e94a223c0a086c4dc6c4432531732475/bin/hmmscan_tblout_to_tsv.py
as you have changed hmmscan to hmmsearch - output table has swapped columns with query and target (I think the same as cmsearch and cmscan does). Please, souble check that hmm postprocessing steps are using correct format. 
For example, https://github.com/EBI-Metagenomics/kegg-pathways-completeness-tool/blob/master/kegg_pathways_completeness/bin/parse_hmmtable/parse_hmmer_tab_separated_table.py#L49 has an option to specify what tool you used 'hmmscan' or 'hmmsearch'. Those fixes are not reflected in that PR but I'm 99% sure you need to change your parsing script for kegg-completeness-tool

Reply:
I will double check, but Biopython handles that internally (the unit tests suggest that https://github.com/biopython/biopython/blob/062eb515f54ff753c96bf53fd10144552075b77a/Tests/test_SearchIO_hmmer3_tab.py#L430)